src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/CompilationTask.kt

```diff
@@ -499,6 +505,27 @@ private fun JvmCompilationTask.updateBuilder(
     it.build()
   }
 
+/**
+ * Copy generated resources from KSP task into generated folder
+ */
+private fun Iterator<String>.copyServicesToGenClasses(directories: Directories): Iterator<String> {
```

Comment:
Thanks for putting this pull request up! Are you able to add a test case or two so that this doesn’t regress in the future?

Reply:
Proguard files are also missing (added https://github.com/bazelbuild/rules_kotlin/issues/991#issuecomment-1657249340), would it make sense to include everything from `MANIFEST_DIR` instead of just `SERVICES_DIR`?

Reply:
Yes, it would. 

It's hard enough to figure out what the plugins are doing, otherwise.

Reply:
would be an easy change then right? just remove the `if ("/$SERVICES_DIR" in it)`  ? or will it contain files we don't want ?